repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/tests/test_modscan.py

```diff
@@ -41,8 +43,17 @@ def test_created_modules(monkeypatch):
         assert not required_modules
 
 
-def test_required_packages():
-    assert set(modscan._REQUIRED_PACKAGES) == set(modscan._create_initram_packages().packages)
+def test_required_packages(monkeypatch):
+    class CurrentActorMocked(object):
```

Comment:
The test fails on the other test - `test_required_packages`.
Fixed file: https://hastebin.com/gifijanimu.py
Also there's no need for using the monkeypatch context. It's necessary only in [certain cases](https://docs.pytest.org/en/latest/reference.html#_pytest.monkeypatch.MonkeyPatch.context) which is I think not this case.